docs/lookup.md

```diff
@@ -0,0 +1,27 @@
+# lookup
+
+This operation adds knodes/kedges and (complete) results. Implementations of this operation are _unique_; their behavior is not fully-specified by the operation parameters.
```

Comment:
Can we add an indication that this is combo operation of `fill+bind+complete` results?